strapi/strapi
# feat(translations): add several items in zh-Hans

### What does it do?

Add several translations for `zh-Hans`.

### Why is it needed?

Current pkg lack of these items.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

eg: 
```
        "global.content-manager": "内容管理",
        "global.plugins": "插件",
        "global.marketplace": "市场",
        "global.settings": "设置",
        "global.profile": "个人资料",
        "content-type-builder.plugin.name": "模型构建器",
```
